src/gui/Connected.qml

```diff
@@ -93,7 +68,7 @@ Item {
         anchors.bottom: deviceControlsGroup.top
 
         property string accessToken: auth.isAuthenticated && Boolean(auth.accessToken) ? auth.accessToken : ""
-        property string studioId: virtualstudio.currentStudio >= 0 ? serverModel[virtualstudio.currentStudio].id : ""
+        property string studioId: virtualstudio.currentStudio.id
```

Comment:
After leaving a studio, my camera indicator light is still on so I suspect we may not be setting `virtualstudio.currentStudio.id` to an empty string somewhere

Reply:
I believe I was able to fix this.